---a/pypy/module/cpyext/include/pytime.h
+++b/pypy/module/cpyext/include/pytime.h
@@ -2,6 +2,7 @@
 #ifndef Py_PYTIME_H
 #define Py_PYTIME_H

+#include <sys/time.h>
 #include "pyconfig.h" /* include for defines */
 #include "object.h"

